README.md

```diff
@@ -16,7 +16,8 @@ Link to [governance](./GOVERNANCE) document.
 ## Meetings
 
 * WG Meeting: [Tuesdays at 11:00am PT (Pacific Time)](https://zoom.us/j/92128676364) (weekly). [Convert to your timezone](https://dateful.com/convert/pt-pacific-time?t=11am).
-  * [Meeting notes and Agenda](https://docs.google.com/document/d/1SVOWQTowigXzbYdorzfa7tMmrcm91yK12LvSONqziJY/edit).
```

Comment:
Do we still have the Google Docs linked somewhere for historical reference?

Reply:
No but I can add it somewhere